### ISSUE TITLE
Docker starts even with 'Start Docker Desktop when you log in' unchecked

### ISSUE THREAD

**Issue by TJSoftware** [docker/for-win#3040]
Sorry for resubmitting this duplicate for other closed items. This is a duplicate of the following items and I used information from those to try to fix .

[GitHub Issue 1](https://github.com/docker/for-win/issues/1991)
[GitHub Issue 2](https://github.com/docker/for-win/issues/476)
[Docker Forum 1](https://forums.docker.com/t/docker-starts-every-time-windows-starts-even-though-start-docker-when-you-log-in-is-not-checked/55576)

### Expected behavior
Docker does not start

### Actual behavior
Docker starts

### Information
Version information when I first noticed
Docker Desktop V 2.0.0.0-win70 (28905)
Docker 18.09.0 build 4d60db4

I ran an update of docker desktop - new version
Docker Desktop V 2.0.0.0-win81 (29211) [NOT resolved]
Docker 18.09.0 build 4d60db4

Uninstalled and downloaded latest Docker Desktop from store.docker.com and reinstalled without using admin running exe.
Docker Desktop V 2.0.0.0-win81 (29211) [NOT resolved]
Docker 18.09.0 build 4d60db4

Uninstalled and reinstalled the previous download running the exe as admin.
Docker Desktop V 2.0.0.0-win81 (29211) [NOT resolved]
Docker 18.09.0 build 4d60db4

### Steps to reproduce the behavior
For each of the versions above I did the following
  1. Verified that "Settings > General > Start Docker Desktop when you log in" is unchecked
  2. Shutdown (not a restart), however this seems to happen on restart as well
  3. Turn on the machine
  4. Watch for docker starting icon

**Comment by WojtekKozaczynski** [comment]
I am having the same issue. Has it been fixed? 

  - The problem is reproducible, and I noticed it after the latest update.
  - Windows Version: Pro 1809. build 17763.134
  - Docker for Windows Version: version 18.09.0, build 4d60db4

**Comment by wdolek** [comment]
@TJSoftware shouldn't this stay opened? I consider @UnstoppableMango's message as workaround, this should be fixed/handled properly by Docker UI/installer itself.

**Comment by UnstoppableMango** [comment]
I would agree. The checkbox to "Start Docker Desktop when you log in" should do _something_. It doesn't seem like it does right now. If this is intended behaviour, their should be some reference to how to properly disable start on boot.

**Comment by JozoSlejko** [comment]
I have the same problem.

Win10 1803 17134.472
Docker version 18.09.0, build 4d60db4

**Comment by mrotteveel** [comment]
Duplicates #2950

**Comment by TJSoftware** [comment]
@wdolek Completely missed you referenced me. I agree it should have remained open. However now that it has been linked to a duplicate that also has more activity and the same info, I will just leave this closed.
